uflash.py

```diff
@@ -69,11 +77,13 @@ def hexlify(script):
     # Convert line endings in case the file was created on Windows.
     script = script.replace(b'\r\n', b'\n')
     script = script.replace(b'\r', b'\n')
+    if can_minify and len(script) > _MAX_SIZE:
```

Comment:
As I mentioned over in the Mu repository, the minification should probably be behind a flag, so it's explicitly asked for. This feels a bit too much like magic, especially since the user may extract the script from the resulting hex file and get a surprise that comments / white-space etc are stripped.